configuration/arm64/docker-compose.yml

```diff
@@ -15,9 +15,7 @@ services:
     network_mode: host
     volumes:
       - "mysql_db:/var/lib/mysql"
-      - ./mysql/install_db.sh:/docker-entrypoint-initdb.d/install_db.sh
-      - ./mysql/localhost.sql:/docker-entrypoint-initdb.d/localhost.sql
-      - ./mysql/script_credentials.sh:/docker-entrypoint-initdb.d/script_credentials.sh
+      - ./mysql/:/docker-entrypoint-initdb.d/
```

Comment:
Please do not remove the existing .sh files. Add your changes to separate .sh file. and add your .sh here

Reply:
I did not remove anything. I just made the volume contain the entire directory instead of the files individually. It didn't make sense to me to have 4 separate volumes containing files that could all be in a single volume.